menas-web/ui/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "menas-web",
+  "version": "2.12.0-AWS-SNAPSHOT",
```

Comment:
Cannot this be taken or updated automatically? I don't see us updating this each time we release new version.

Reply:
There was at a certain point a sync tool, I should try it again